test/DrawBBox.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python2
+import subprocess 
+import numpy
+import sys
+
+# Edit this as required (windows, mac).  Be able to write this.
+PLY_FILE="/tmp/displaz_bbox.ply"
+
+# write .ply file representing a box with specfied center and transform to f
+def MakeBBox(v_center, transform, f):
+  f.write("ply\n")
+  f.write("format ascii 1.0\n")
+  f.write("comment simple rectangular prism\n")
+  f.write("element vertex 8\n")
+  f.write("property float x\n")
+  f.write("property float y\n")
+  f.write("property float z\n")
+  f.write("element face 2\n")
+  f.write("property list uint int vertex_index\n")
+  f.write("end_header\n")
+
+  # apply transform to unit vectors and then add result to center
+  unit_vectors = [ 
+    [0, 1, -1, 0], [1, 0, -1, 0], [0, -1, -1, 0], [-1, 0, -1, 0], 
+    [0, 1, 1, 0], [1, 0, 1, 0], [0, -1, 1, 0], [-1, 0, 1, 0]
+  ] 
+  xfm_unit_vectors = [transform.dot(unit) for unit in unit_vectors]
+  vertices = [numpy.add(v_center, xuv) for xuv in xfm_unit_vectors]
+
+  # write out resulting vertices
+  map(lambda v: f.write("%s\n" % " ".join(str(elem) for elem in v)), vertices)
+  f.write("4 0 1 2 3\n")
+  f.write("4 4 5 6 7\n")
+
+# write out the .ply file and have Displaz render it
+def UpdateDisplaz(v, T):
+    t = open(PLY_FILE, 'w')
+    t.truncate()
+    MakeBBox(v, T, t)
+    t.close()
+    p = subprocess.Popen(["displaz -script -rmtemp " + PLY_FILE], stdout=subprocess.PIPE, shell=True, bufsize=1)
```

Comment:
Also, how about labelling your dataset?  If you use the `-label Box` option, it will show up in the dataset view with a more readable name, rather than the temporary file name.